src/i18n/it.js

```diff
@@ -0,0 +1,43 @@
+export default {
+    header: {
+        slogan: 'Il futuro dello sviluppo web.'
+    },
+    callToAction: {
+        start: 'Per cominciare',
+        join: 'Unisciti alla chat'
+    },
+    nav: {
+        code: 'Code',
+        cloud: 'Cloud',
+        docs: 'Documentazione'
+    },
+    bio: {
+        first: Architettura non bloccante, basata sugli eventi grazie alla libreria <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> di Apple che garantisce <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">alte prestazioni</a>.',
+        second: 'Scritto in <a target="_blank" href="https://swift.org">Swift</a>, il linguaggio di programmazione che è potente e facile da imparare.',
```

Comment:
Wouldn't 'il linguaggio di programmazione potente e facile da imparare' flow better even if is not strictly the same phrase?

Reply:
For sure!